templates/partial/filter.html

```diff
@@ -18,18 +19,29 @@ <h2 class="govuk-heading-m">Selected filters</h2>
             <div class="moj-filter__heading-action">
               <p>
                 <a class="govuk-link govuk-link--no-visited-state"
-                href="{% url 'home:search' %}{% query_string domains=None %}"
+                href="{% url 'home:search' %}{% query_string domain=None classifications=None where_to_access=None %}"
```

Comment:
Suggestion: Could we hide selected filters entirely if there are no selected filters? (Not sure if the component is designed to be used this way, but noticed it looks odd having a clear filters button with no filters)

Reply:
Selected filters is now hidden when there are no filters present.